src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -358,35 +349,6 @@ function BaseReportActionContextMenu({
     );
 }
 
-export default withOnyx<BaseReportActionContextMenuProps, BaseReportActionContextMenuOnyxProps>({
-    betas: {
-        key: ONYXKEYS.BETAS,
-    },
-    reportActions: {
-        key: ({originalReportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`,
-        canEvict: false,
```

Comment:
```
canEvict: false,
```

I don't see this option is applied when using useOnyx. Are we missing anything here?